server/src/main/java/org/opensearch/indices/ingest/package-info.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/** Indices ingestion module package. */
+package org.opensearch.indices.ingest;
```

Comment:
The term "ingest" is definitely overloaded. `_bulk` is a type of ingestion, there are ingest pipelines, etc. I'd suggest using `polling.ingest` or `pollingingest` or anything else that helps disambiguate this area of the code from the ingest related pieces.

Reply:
Agreed. `pollingingest ` sounds good to me.